identity/identity.go

```diff
@@ -74,6 +74,11 @@ func (c CentID) BigInt() *big.Int {
 	return utils.ByteSliceToBigInt(c[:])
 }
 
+// Bytes returns CentID in bytes
+func (c CentID) Bytes() []byte {
```

Comment:
You can always use `c[:]` to get the bytes here. We are using this multiple places.

check out
`coredocument.go#PrepareNewVersion` function